content/docs/command-reference/build.md

```diff
@@ -6,11 +6,16 @@ Python package.
 ## Synopsis
 
 ```usage
-usage: mlem build [options] model [subtype]
+Usage: mlem build builder [options] model
```

Comment:
let's keep it unified across docs Usage -> usage

Reply:
fixed